### PR TITLE
Bug Fix #1449

### LINKED ISSUE #1449: No opportunity to change JSQMessagesInputToolbar preferredDefaultHeight
It doesn't seem like you can change the height of the `inputToolbar` using the documented property `preferredDefaultHeight` since it is used to configure the height constraint in `viewDidLoad`. Am I missing something or is this an actual issue. Thanks.

### PR DESCRIPTION
**Pull request by GuyKahlon**
Hey Jesse, 

## Pull request checklist

- [ ] This fixes issue #1449.
- [Yes] All tests pass. Demo project builds and runs.
- [No merge conflicts] I have resolved any merge conflicts.
- [Yes] I have squashed my commits into 1 commit.
- [Yes] I have followed the [coding style](https://github.com/jessesquires/HowToContribute#style-guidelines), and reviewed the [contributing guidelines](https://github.com/jessesquires/JSQMessagesViewController/blob/develop/.github/CONTRIBUTING.md). Confirmation: ____

## What's in this pull request?
Fix very small bug from the open issues. 

Thanks,
Guy

### CODE REVIEW

File: JSQMessagesViewController/Controllers/JSQMessagesViewController.m [hunk 1]
```diff
@@ -196,7 +196,8 @@ - (void)viewWillAppear:(BOOL)animated
     [super viewWillAppear:animated];
     [self.view layoutIfNeeded];
     [self.collectionView.collectionViewLayout invalidateLayout];
-
+    self.toolbarHeightConstraint.constant = self.inputToolbar.preferredDefaultHeight;
```
Comment (jessesquires) [inline JSQMessagesViewController/Controllers/JSQMessagesViewController.m]: we should probably put this before `layoutIfNeeded` above, right?

**Comment by jessesquires** [comment]
Thanks @GuyKahlon ! Looks good.

If you can, could we remove commit 701fd26 from this? (seems like an accident)

**Comment by GuyKahlon** [comment]
Hey @jessesquires,

Commit 701fd26 has been removed.
Move update toolbarHeightConstraint.constant property before layoutIfNeeded.

**Comment by jessesquires** [comment]
:+1: 

last thing, mind squashing these commits into 1? :smile:

**Comment by GuyKahlon** [comment]
Hey Jesse,

I tried to squash the committees into 1 with rebase (http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html) but it looks worse now, we have a 5 comments after the merge.

Do you prefer I'll send you  a new Pull Request with just 1 commit? 
Or maybe you know a better way for squashing.


Sorry about all the mess 😁

**Comment by jessesquires** [comment]
@GuyKahlon -- sometimes a new pull request is easiest :grin:

**Comment by jessesquires** [comment]
subsumed by #1477

**Pull request closed without merging**